lib/mix/tasks/docs.ex

```diff
@@ -200,6 +204,37 @@ defmodule Mix.Tasks.Docs do
   the first will take precedence. Functions that don't have a custom group will
   be listed under the default "Functions" group.
 
+  ## Additional javascript script
+
+  Since version `0.19.4` ExDoc includes a way to enrich the documentation
```

Comment:
I think since we are adding a new feature, this shouldn't be a PATCH release, but a MINOR one, so I would change the version to `0.20.0`